it/apps/src/main/content/jcr_root/apps/forms-core-components-it/samplepages/loadunloadaf/v1/loadunloadaf/loadunloadaf.html

```diff
@@ -0,0 +1,43 @@
+<div class="custom-embed">
+        <script>
+            
+           function loadform(btn) {
+             var formPath = btn.getAttribute('data-form-path')
+             loadAdaptiveForm({
+               formPath: formPath,
+               CSS_Selector: '.customafsection'
+             })
+           }
+
+           var loadAdaptiveForm = function(options) {
+             var path = options.formPath + '.html?wcmmode=disabled';
```

Comment:
Can you write a comment that on publish instance the query parameters are not required